frontend/app/components/op-context-menu/op-context-menu.html

```diff
@@ -11,7 +11,7 @@
            tabindex="0"
            [ngClass]="{ 'inactive': item.disabled }"
            [attr.href]="item.href || ''"
-           [attr.disabled]="item.disabled"
+           [attr.aria-disabled]="item.disabled"
```

Comment:
Shouldn't we still set the `[disabled]` attribute for styling?

Reply:
It should not have any effect on the styling since `disabled` is not supported for a-tags anyway.